pkgs/applications/networking/zerobin/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, python3Packages
+, pythonOlder
+, buildPythonApplication
+, fetchFromGitHub
+, nodePackages
+, doit
+}:
+
+buildPythonApplication rec {
```

Comment:
Here there should be `python3.pkgs.buildPythonApplications`.

Reply:
Oops, sorry I should have tested the build before uploading. Because the replacing of `python3Packages.callPackage` by `callPackage` in last change, `pythonOlder` needed to be imported from `python3Packages`.
Note that instead of `python3.pkgs` I'm using the `python3Packages` alias, just because it's what is suggested in https://nixos.org/manual/nixpkgs/stable/#python-library-packages-in-nixpkgs